pkg/api/validation/validation_test.go

```diff
@@ -6130,16 +6093,6 @@ func TestValidateNode(t *testing.T) {
 				Name:   "abc",
 				Labels: validSelector,
 			},
-			Status: api.NodeStatus{
```

Comment:
To reviewer: NodeStatus isn't validated in `ValidateNode`, so we don't need it here.